followthemoney/schema/CallForTenders.yaml

```diff
@@ -0,0 +1,122 @@
+CallForTenders:
+  label: "Call for tenders"
+  plural: "Calls for tenders"
+  description: >
+    A public appeal issued by an authority, possibly on behalf of another,
+    for buying a specific work, supply or service
+  extends:
+    - Thing
+    - Interval
+  matchable: false
+  required:
+    - title
+    - authority
+  featured:
+    - title
+    - authority
+  caption:
+    - title
+  properties:
+    callId:
+      label: "CfT unique id"
+      type: identifier
+    # What is "CA"? Contract award? Maybe it should then live there...
+    # cftCaUid:
+    #   label: "CfT CA unique id"
+    #   type: identifier
+    title:
+      label: "Title"
+    authority:
+      label: "Name of contracting authority"
+      reverse:
+        name: callForTenders
+        label: "Call For Tenders"
+      type: entity
+      range: LegalEntity
+    onBehalfOf:
+      label: "Published on behalf of"
+      reverse:
+        name: delegatedCallForTenders
+        label: "Delegated call for tenders"
+      type: entity
+      range: LegalEntity
+    publicationDate:
+      label: "Date of publication/invitation"
+      type: date
+    evaluationMechanism:
+      label: "Evaluation mechanism"
+    procurementType:
+      label: "Procurement type"
+    directive:
+      label: "Directive"
+    procedure:
+      label: "Procedure"
+    # What does this mean?
+    # involves:
+    #   label: "Call for tenders involves"
```

Comment:
This is is a field that's used to differentiate between different types of CfTs. It seems to be a kind of bodge to use the CfT system for other purposes. Here's the label used for it on the PDF version of the Contract Notice: 

<img width="381" alt="image" src="https://user-images.githubusercontent.com/282964/199509069-bc443a4d-248c-4534-a954-04628474d4b1.png">

In this Legal Notice [178 of 2005](https://contracts.gov.mt/en/Resources/Documents/LN178_2005.pdf)  (page 9) you have definitions of DPS and framework agreement.